apps/fiona/fiona.cpp

```diff
@@ -81,9 +81,15 @@
     // Thus, we do not enable it in this case.
         #define _GLIBCXX_PARALLEL
     #endif  // #if !defined(__MINGW32__) || defined(__amd64__) || defined(__x86_64__) || defined(__ia64__)
+
+    #if defined(__INTEL_COMPILER)
+        #warning "The Intel Compiler crashes with _GLIBCXX_PARALLEL defined (at least until v16.0.2). Therefore, the feature will be disabled."
+        #if __INTEL_COMPILER_BUILD_DATE <= 20160204
```

Comment:
Is COMPILER_BUILD_DATE the right macro to check version? I know that for GCC it is not, because 5.3.2 might be released after 6.0.0...